tests/mysql/WP_MySQL_Server_Suite_Parser_Tests.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Parser tests using the full MySQL server test suite.
+ */
+class WP_MySQL_Server_Suite_Parser_Tests extends TestCase {
+	const TEST_DATA_PATH = __DIR__ . '/data/queries.csv';
```

Comment:
How about a more descriptive file name?

```suggestion
	const TEST_DATA_PATH = __DIR__ . '/data/mysql-server-unit-test-queries.csv';
```

Reply:
@adamziel I renamed it to `mysql-server-tests-queries.csv`. I think they are not really `unit` tests, it's more like end-to-end test suite, I guess.